generators/client/templates/react/webpack/webpack.common.js.ejs

```diff
@@ -111,24 +109,13 @@ module.exports = options => merge(
   },
   plugins: [
     new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: `'${options.env}'`,
```

Comment:
I think React and some libs like Redux for example internally still use `process.env.NODE_ENV` for certain logic and hence we should ensure its value is passes as well IMO, unless its automatically set

Reply:
I don't know Redux.
Use of `process.env` should be avoided: https://github.com/webpack/changelog-v5/blob/master/MIGRATION%20GUIDE.md#level-5-runtime-errors.

Reply:
Ya but many React frameworks might use it s its better to make sure its set. Redux definitely does it for enabling dev tools and stuff

Reply:
According to https://webpack.js.org/guides/production/#specify-the-mode.
NODE_ENV is set by webpack when mode is set, so no need to set it manually.